backend/cmd/migrate/migrations/000009_experimentation_id_column_type_change.up.sql

```diff
@@ -0,0 +1,5 @@
+alter table experiment_run alter column "id" SET DATA type varchar(100) using id::varchar;
+alter table experiment_run alter column "experiment_config_id" SET DATA type varchar(100) using experiment_config_id::varchar;
+ALTER TABLE experiment_run ADD PRIMARY KEY (id);
+alter table experiment_config alter column "id" SET DATA type varchar(100) using id::varchar;
```

Comment:
we do not need per se but I thought that it would be a good idea to change it so that it follows the same convention the `experiment_run` table does - use `string` as an `id` instead of `BIGINT`.